libraries/AP_HAL_ChibiOS/hwdef/kha_eth/defaults.parm

```diff
@@ -0,0 +1,33 @@
+# Network 1
+SERIAL1_PROTOCOL 0
+SERIAL1_BAUD 230400
+SERIAL1_OPTIONS 8
+
+# Network 2
+SERIAL2_PROTOCOL 0
+SERIAL2_BAUD 230400
+
+# ------------
+# Networking
+# ------------
+NET_ENABLED      1
+NET_DHCP         0
+
+NET_IPADDR0      172
```

Comment:
Consider adding a number 4 to indicate this is IPv4, in the eventuality that IPv6 is supported. 

`NET_IP4ADDR0` and `NET_IP6ADDR0` are now clear. 

Rather than:
`NET_IPADDR0` and `NET_IP6ADDR0`

Reply:
Good point. Ipv6 is indeed supported right now but you can't set an ipv6 ip. I'll apply you suggestion.

Reply:
done